gatsby-node.js

```diff
@@ -121,7 +121,7 @@ exports.createPages = async ({ graphql, actions }) => {
   const allVotelogs = votelogs.data.allVotelogYaml.edges
   const votelogPerPage = 4
   const numPages = Math.ceil(allVotelogs.length / votelogPerPage)
-  Array.from({ length: numPages }).forEach((_, i) => {
+  Array.from({ length: numPages}).forEach((_, i) => {
```

Comment:
Unnecessary removal of space. 

This is weird be cause prettier should be run on every commit 

```js
// package.json
{
  // ...
  "husky": {
    "hooks": {
      "pre-commit": "lint-staged"
    }
  },
  "lint-staged": {
    "src/**/*.{js,json,css}": [
      "prettier --write",
      "git add"
    ],
    // ...
  }
}
```

Reply:
I see, that's because `gatsby-node.js` is in root and pattern is `"src/**/*.{js,json,css}"`. Will fix to `"**/*.{js,json,css}"` in another issue.